tools/CouchDB_Import_Demographics.php

```diff
@@ -135,7 +167,7 @@ function _generateQuery() {
             $EDCFields = ", c.EDC as EDC";
             $fieldsInQuery .= $EDCFields;
         }
-        $concatQuery = $fieldsInQuery . $tablesToJoin . " WHERE s.Active='Y' AND c.Active='Y' AND c.Entity_type != 'Scanner'";
+        $concatQuery = $fieldsInQuery . $tablesToJoin . " WHERE s.Active='Y' AND c.Active='Y' AND c.Entity_type != 'Scanner'"." GROUP BY s.ID";
```

Comment:
I tried on mysql 5.7, also tried it from the import script... never got this error
@driusan any ideas?